### PR TITLE
Remove description of certificate alert

### PR DESCRIPTION
**Pull request by shadinaif**
Related to: https://edraakteam.slack.com/archives/GG2D43MPB/p1583929061009200

### CODE REVIEW

File: lms/djangoapps/courseware/date_summary.py [hunk 1]
```diff
@@ -366,18 +366,7 @@ def register_alerts(self, request, course):
...
-                )).format(
-                    time_remaining_string=self.time_remaining_string,
-                    learner_profile_link=HTML(
-                        '<a href="{learner_profile_url}">{learner_profile_name}</a>'
-                    ).format(
-                        learner_profile_url=reverse('learner_profile', kwargs={'username': request.user.username}),
-                        learner_profile_name=_('Learner Profile'),
-                    ),
-                ),
+                '',  # Edraak: Message description deleted
```
Comment (ghost) [inline lms/djangoapps/courseware/date_summary.py]: Either

```suggestion
                Text(''),  # Edraak: Message description deleted
```

or with a space

```suggestion
                Text(' '),  # Edraak: Message description deleted
```

**Review by ghost — APPROVED** [review]
Please fix the failed tests or mark them as expected failures (`@expectedFailure`) like below. LGTM otherwise.

https://github.com/Edraak/edraak-platform/blob/19bf6f4ebc6ea8c47af11bf87536806a7abe7f73/openedx/core/djangoapps/auth_exchange/tests/mixins.py#L104-L107

File: lms/djangoapps/courseware/date_summary.py (hunk 1 above)
Comment (shadinaif) [reply lms/djangoapps/courseware/date_summary.py]: done

Comment (shadinaif) [reply lms/djangoapps/courseware/date_summary.py]: done

**Pull request merged**